sdv/constraints/tabular.py

```diff
@@ -479,7 +479,9 @@ def _transform(self, table_data):
 
         diff_column = high - low
         if self._is_datetime:
+            nat_mask = np.isnan(diff_column)
```

Comment:
I think you should just call it `nan_mask`, I don't think it's obvious that `nat` stands for `NaT`.